Index: app/src/user.InterfaceLayer/Pages/Last.page/ui/LastPage.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import React, { FC } from \"react\";\r\n\r\nconst LastPage: FC = () => <div> SvyatoslavZhilinLastPage </div>;\r\n\r\nexport default LastPage;\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/user.InterfaceLayer/Pages/Last.page/ui/LastPage.tsx b/app/src/user.InterfaceLayer/Pages/Last.page/ui/LastPage.tsx
--- a/app/src/user.InterfaceLayer/Pages/Last.page/ui/LastPage.tsx	
+++ b/app/src/user.InterfaceLayer/Pages/Last.page/ui/LastPage.tsx	
@@ -1,5 +1,13 @@
 import React, { FC } from "react";
+import DataTimePicker from "user.InterfaceLayer/Libraries/main.library/src/UI_KIT/Atoms/DataTimePicker.atom";
 
-const LastPage: FC = () => <div> SvyatoslavZhilinLastPage </div>;
+const LastPage: FC = () => (
+	<div>
+		<DataTimePicker
+			onChange={() => {}}
+			selected={new Date()}
+		/>
+	</div>
+);
 
 export default LastPage;
